### PR TITLE
Added MindsDB to the Swag list

### PR DESCRIPTION
**Pull request by amitmishra11**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [X] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [X] I'm not adding a company from the blocklist
3. [X] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

**Comment by amitmishra11** [comment]
@crweiner  I tried removing the conflicts but am not able to remove flags that codeclimate  puts in.
please see

File: docs/list.md [hunk 1]
```diff
@@ -154,10 +155,32 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
-- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues){:target="\_blank"}.
-- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/){:target="\_blank"} for more info.
+<<<<<<< HEAD
+    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md) for more details.
+- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f) and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X) to connect with Meilisearch employees and for any queries.
+- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues).
+- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/) for more info.
+>>>>>>> e2b3adf2093b5a410fba097c66210c512628e5e4
```
Comment (crweiner) [inline docs/list.md:163]: It looks like you have a merge conflict here. Please remove the duplicate text and keep the ones that have `{:target="\_blank"}` after the links.

File: docs/list.md [hunk 2]
```diff
@@ -154,10 +155,32 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+
+
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:165]: The Markdown linter in CodeClimate is looking for 1 paragraph space, not 2, between each header.

File: docs/list.md [hunk 3]
```diff
@@ -135,6 +135,7 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### M
 
+
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:138]: The Markdown linter in CodeClimate is looking for 1 paragraph space, not 2, between each header.

File: docs/list.md [hunk 4]
```diff
@@ -154,10 +155,32 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues).
+- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/) for more info.
+>>>>>>> e2b3adf2093b5a410fba097c66210c512628e5e4
+
+
+#### **MindsDB**
+
+- **Swag**:Razer Blade 15 Laptop,T-shirt, cool audio gadgets, and cash prize
+- **Requirements**:
+    - For every 10 points, you will receive one entry into the draw for the awesome Razer Blade 15 Laptop powered by the top NVIDIA GeForce RTX 3080 Ti GPU, 14-core Intel i9 CPU and more…
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:170]: It's a small thing, but please make sure there aren't any stray spaces, tabs, paragraph breaks, line breaks, etc. Keeping these stray whitespace marks in can affect the Markdown -> HTML conversion during CI/CD building.

File: docs/list.md [hunk 5]
```diff
@@ -154,10 +155,32 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Swag**:Razer Blade 15 Laptop,T-shirt, cool audio gadgets, and cash prize
+- **Requirements**:
+    - For every 10 points, you will receive one entry into the draw for the awesome Razer Blade 15 Laptop powered by the top NVIDIA GeForce RTX 3080 Ti GPU, 14-core Intel i9 CPU and more… 
+    You can also win one of the 10 gadgets of your choice: 
+    JBL Earbuds or Rugged Outdoor Speaker.Entries will close at midnight (PST) Monday, 31st of October with the winner announced on the first week of November.
+    To participate, you need to have your GitHub PRs merged and complete the form with your details.
+    - Share a well-described idea of a new MindsDB integration. The authors of top upvoted ideas will get a T-shirt, cool audio gadgets, and a cash prize::
+    1st place: $500
+    2nd place: $300
+    3rd place: $200
```
Comment (crweiner) [inline docs/list.md:177]: Make sure that you follow Markdown formatting for unordered lists for here and lines 171 - 173 as well.

Comment (crweiner) [reply docs/list.md:177]: [This site may help](https://www.markdownguide.org/basic-syntax/#unordered-lists)

**Review by crweiner — CHANGES_REQUESTED** [review]
Hey @amitmishra11! 👋 Thanks for the new PR! Just a few things to adjust, please. We've got to keep the bots happy before they decide to revolt. 🤖 

Please let me know if you need help with the merge conflict or markdown formatting! I'd like to give you a chance to learn and try on your own!

Be sure to check the [Netlify deploy preview ](https://deploy-preview-582--hacktoberfest-swag-list.netlify.app/list)to make sure that things look the way you'd like. Please ping me again if you need any help. 👍

File: docs/list.md [hunk 6]
```diff
@@ -150,14 +150,22 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Swag**:Razer Blade 15 Laptop,T-shirt, cool audio gadgets, and cash prize
 - **Requirements**:
-    - Submit 1 or more pull requests to qualify as a contributor. Check out the [Contributing Guidelines](https://github.com/meilisearch/.github/blob/main/Hacktoberfest_2022_contributors_guidelines.md){:target="\_blank"} for more details.
-- **How to sign up**: Join the [Slack community](https://www.notion.so/6cd1509802984122ab74745f637b694f){:target="\_blank"} and the [#hacktoberfest channel](https://meilicommunity.slack.com/archives/C02G1H43J2X){:target="\_blank"} to connect with Meilisearch employees and for any queries.
-- **Issues**: See the list of [hacktoberfest-labeled issues](https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&state=open&type=Issues){:target="\_blank"}.
-- **Notes**: Check out the [blog post](https://blog.meilisearch.com/contribute-hacktoberfest-2022/){:target="\_blank"} for more info.
+    - For every 10 points, you will receive one entry into the draw for the awesome Razer Blade 15 Laptop powered by the top NVIDIA GeForce RTX 3080 Ti GPU, 14-core Intel i9 CPU and more. You can also win one of the 10 gadgets of your choice:
```
[the remark this answers is not in the document]
Comment (amitmishra11) [reply docs/list.md:157]: @crweiner  Thank you so much for the help! Telling me about each issue separately. With that help, I was able to knock 9 out of 10 issues in the previous commit. Just this one is left

[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md]: You did a really, really good job! Thank you very much for tackling the issues that CodeClimate flagged! I appreciate it and I hope it taught you something along the way, @amitmishra11!

**Review by crweiner — APPROVED** [review]

**Comment by crweiner** [comment]
Hello again @amitmishra11! 👋 Thank you so much for adding MindsDB to the 2022 Hacktoberfest Swag List! 🎉  I loved seeing how you took my review into consideration and made some adjustments to your PR. I hope that this was a good learning experience for you! This should give you a boost in confidence and a great start to Hacktoberfest! 

Let's send some great people to MindsDB so they can receive some helpful PRs! 🚀 

you rock

Thanks again, and happy Hacktoberfest! 🎃

**Pull request merged**